PostResponses/PostResponsesVDRs.js

```diff
@@ -1,21 +1,30 @@
 if (fromVendor){
-  // This step is needed to ensure that nothing else is executed if you're performing a simple GET request. 
-  done(transformedObject)
+  // This step is needed to ensure that nothing else is executed if you're performing a simple GET request.
```

Comment:
@kiawnna that's not entirely true.  If this if the statement were omitted, the VDR would function correctly, assuming there is not any code that needs to run for a GET or GET by ID call.  So I would update this to say something like "Using this if block, the response from the vendor can be manipulated via code".  The if block is only needed if the user needs to differentiate between data that is coming/going to the vendor and the response from those vendor calls.

Reply:
@nick-bair That comment was in there originally so I left as is, but I thought the same thing! Updated.